File: python/src/interface/menus.py
# interface/menus.py


from utils import limpar_tela


def menu_principal():
    limpar_tela()
    print("╔════════════════════════════════════════════╗")
    print("║          Bem-vindo à Biblioteca!           ║")
    print("╠════════════════════════════════════════════╣")
    print("║              1. Ver catálogo               ║")
    print("║              0. Sair                       ║")
    print("╚════════════════════════════════════════════╝")


def menu_escolha_livro(total):
    print(f"1 - {total}. Escolher livro")
    print("0. Voltar")


def menu_livro_disponivel():
    print("1. Requisitar")
    print("2. Adicionar avaliação")
    print("0. Voltar")


def menu_livro_emprestado():
    print("1. Devolver")
    print("2. Adicionar avaliação")
    print("0. Voltar")

File: javascript/interface/menus.js
// interface/menus.js


function menuPrincipal() {
    console.clear();
    console.log("╔════════════════════════════════════════════╗");
    console.log("║          Bem-vindo à Biblioteca!           ║");
    console.log("╠════════════════════════════════════════════╣");
    console.log("║              1. Ver catálogo               ║");
    console.log("║              0. Sair                       ║");
    console.log("╚════════════════════════════════════════════╝");
}

function menuEscolhaLivro(total) {
    console.log(`1 - ${total}. Escolher livro`);
    console.log("0. Voltar");
}

function menuLivroDisponivel() {
    console.log("1. Requisitar");
    console.log("2. Adicionar avaliação");
    console.log("0. Voltar");
}

function menuLivroEmprestado() {
    console.log("1. Devolver");
    console.log("2. Adicionar avaliação");
    console.log("0. Voltar");
}

module.exports = {
    menuPrincipal,
    menuEscolhaLivro,
    menuLivroDisponivel,
    menuLivroEmprestado
};
